app/configurations/config.hsl.js

```diff
@@ -491,7 +491,7 @@ export default {
         timeBeforeSurcharge: 120 * 60,
         showRentalStations: true,
       },
-      bolt: {
+      bolt_helsinki: {
```

Comment:
I guess this can be now plain bolt , as the network covers other cites too. Close thye respective otp data PR.